Automattic/wp-calypso
# Ads: ES6ify earnings form

This PR ES6ifies the ads earnings form and removes all ESLint warnings from it. This will help us do any further modifications without much changes while working on the upcoming ads modifications - https://github.com/Automattic/wp-calypso/milestone/169

There should be no functional, visual or behavioral changes.

To test:
* Checkout this branch
* Go to `/ads/earnings/$site` where `$site` is a Jetpack site with a Premium or Professional plan.
* Verify the earnings are properly listed and there are no visual/behavioral changes.
* Play with the info toggles and verify they work like they did before.
* Switch tabs, change some earnings settings, go back and verify everything works properly.
* Test the above with a WordPress.com site and verify there are no regressions.
